pkg/neco/cmd/init_local.go

```diff
@@ -109,3 +133,32 @@ func issueCerts(ctx context.Context, vc *api.Client, commonName, cert, key strin
 	return neco.WriteFile(key, secret.Data["private_key"].(string))
 
 }
+
+func issueSabakanTLSCerts(ctx context.Context, vc *api.Client, cert, key string) error {
```

Comment:
Please make it clear that this is for server certificates.

```suggestion
func issueSabakanServerCerts(ctx context.Context, vc *api.Client, cert, key string) error {
```